D/DecFP/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+using BinaryBuilder
+
+name = "DecFP"
+version = v"2.0U2" # 2.0 Update 2
```

Comment:
Can we drop the `U2` bit?  This is not going to play nicely with the package manager:
```suggestion
version = v"2.0"
```

Reply:
The Previous release was 2.0U1 so I could go with v"2.0.2".  I was just trying to keep it as close as possible to the version number in the distribution file name.  If this will cause problems, I'll change it.

Reply:
In general we do try to keep the upstream version number as long as it is in the form `major.minor.patch`.  Other numbers are going to be troublesome.  If you think that that could be translated to `2.0.2`, that's a good option